app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -47,14 +47,15 @@ def monitor_events
     reset_event_monitor_handle
   end
 
-  def process_event(event)
+  def queue_event(event)
     event_data = extract_event_data(event)
-    if event_data.nil?
-      _log.debug "#{log_prefix} Discarding event [#{event_data}]"
-    else
-      _log.info "#{log_prefix} Queuing event [#{event_data}]"
-      EmsEvent.add_queue('add_kubernetes', @cfg[:ems_id], event_data)
-    end
+    _log.info "#{log_prefix} Queuing event [#{event_data}]"
+    EmsEvent.add_queue('add_kubernetes', @cfg[:ems_id], event_data)
+  end
+
+  def filtered?(event)
+    event_data = extract_event_data(event)
```

Comment:
could be just `extract_event_data(event).nil?` or `!extract_event_data(event)`

also this reveals something for later refactoring.
in this case the raw event data would be parsed twice.
so maybe it makes sense to introduce a proper event object that is passed around, not sure